Create podcast.md (#4665)
sourcegraph/about: handbook/marketing/content/web-series.md

## Web series

> Describe what a web series is and why we do them at Sourcegraph

Reviewer: Is this a left-over TODO or still valid TODO? If it's the latter I'd hide it in a comment:

```suggestion
<!-- Describe what a web series is and why we do them at Sourcegraph -->
```

## Dev Tool Time

> Dev Tool Time is a bi-weekly web series livestreamed on Twitch where Sourcegraph engineers interview other devs in the community about their desk set up, favorite dev tools, productivity hacks, and more.

Reviewer: I'd definitely link to previous examples so people can get a better idea of what it's about (and it's also the first think I'd look for if I'd land on that page as a guest)

For example:

```suggestion
Dev Tool Time is a bi-weekly web series livestreamed on [Twitch](https://www.twitch.tv/sourcegraph) where Sourcegraph engineers interview other devs in the community about their desk set up, favorite dev tools, productivity hacks, and more. Recordings are later available on our [YouTube account](https://www.youtube.com/watch?v=W7DHo90Boa4&list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF).
```

But can be phrased differently, or with both links in a separate paragraph:

```
- Our Twitch account: [twitch.tv/sourcegraph](https://www.twitch.tv/sourcegraph)
- YouTube playlist with all previous episodes: [Dev Tool Time on YouTube](https://www.youtube.com/watch?v=W7DHo90Boa4&list=PL6zLuuRVa1_iDEP4EicZ8972RgyccCRGF)
```

> This series aims to help new developers learn about existing tools but also update experienced developers who might not know about the latest developer tools. While Sourcegraph may be mentioned, it will not be the main focus. This series is genuinely to inform developers about all the different tools out there and enhance developer awareness about the importance of having the right tools setup to increase coding efficiency.

Reviewer: I'd personally remove the "latest", because showing off new things is not a focus, IMHO.

```suggestion
This series aims to help new developers learn about existing tools but also update experienced developers who might not know about certain developer tools and workflows. While Sourcegraph may be mentioned, it will not be the main focus. This series is genuinely to inform developers about all the different tools out there and enhance developer awareness about the importance of having the right tools setup to increase coding efficiency.
```

> ### What makes a great Dev Tool Time guest?

Reviewer 2: @tjdevries @mrnugget, would love your thoughts on this.

Explain what we are looking for (Actively develops, interesting dev environment, new tools to show off, productivity hacks to show, makes tools)

1. How to propose a new guest
2. [How to apply to become a guest](#Apply-to-become-a-guest)

#### Show format

There will be a set structure that we apply to each event to create a consistent flow and to ensure each event is filled with great in-depth content. During the series, a Sourcegraph host will be talking to an external guest who shares and showcases their preferred developer tools.

- Host: (what does this role entail)
- Co-host: (what does this role entail)
- Moderator: (what does this role entail)

##### Run of show

- 10 minutes prior to start time: Hosts & guests enter green room
- 1 minutes prior to start time: Moderator preps guest to go live (quiet the room, mute, etc.)
- 30 seconds prior to start time: Moderator leads count down to going live and gives host signal to start
- Host opens the show
- Host leads intros, banter, background on guest
- Guest shows hardware setup (guest shares photo of desk)
- Guest dives into the tools, demos, live showcasing tools (guest shares their screen)

> - Moderator shares prepares and shares questions through out the show

Reviewer: ```suggestion
- Moderator prepares and shares questions from chat through out the show
```

- Moderator signals 5 minute mark to wrap up show at 55 minute mark
- Host leads outro, where to find more information, helpful links, teases next guest

### Program details

- When: The second and fourth Wednesday of every month at 11:00am PT / 6:00 PM UTC
- Where: [Twitch](https://www.twitch.tv/sourcegraph/)

> - Host: TBD TJ DeVries / Erika Heidi

Author: ```suggestion
- Hosts: TJ DeVries, Thorsten Ball 
```

> - Rotating hosts: Thorsten Ball, Kelli Rockwell, Beyang Liu

Author: ```suggestion

```

Author: @jennybergen @tjdevries @mrnugget I suggest removing this and adding something about `guest host` and how to sign up to become one. This leaves it open for anyone in the company to guest host
